src/components/pipelines-overview/list-pages/PipelineRunsForRepositoriesRow.tsx

```diff
@@ -1,23 +1,42 @@
 import * as React from 'react';
-import { mainDataType } from '../utils';
-import { RowProps } from '@openshift-console/dynamic-plugin-sdk';
+import { SummaryProps, getReferenceForModel } from '../utils';
+import { ResourceLink, RowProps, useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import { formatTime, formatTimeLastRunTime } from '../dateTime';
+import { ALL_NAMESPACES_KEY } from '../../../consts';
+import { RepositoryModel } from '../../../models';
 
-const tableColumnClasses = ['', '', '', '', '', '', ''];
+const tableColumnClasses = [
+  '', //name
+  '', //namespace
+  'pf-m-hidden pf-m-visible-on-md', //total plr
+  'pf-m-hidden pf-m-visible-on-md', //total duration
+  'pf-m-hidden pf-m-visible-on-xl', //avg duration
+  'pf-m-hidden pf-m-visible-on-xl', //success rate
+  'pf-m-hidden pf-m-visible-on-xl' //last run time
+];
+const repositoryReference = getReferenceForModel(RepositoryModel);
 
-const PipelineRunsForRepositoriesRow: React.FC<RowProps<mainDataType>> = ({
+const PipelineRunsForRepositoriesRow: React.FC<RowProps<SummaryProps>> = ({
   obj,
 }) => {
+  const [activeNamespace] = useActiveNamespace();
+  
   return (
     <>
-      <td className={tableColumnClasses[0]}>{obj.repoName}</td>
-      <td className={tableColumnClasses[1]}>{obj.projectName}</td>
+      <td className={tableColumnClasses[0]}>
+        <ResourceLink
+      kind={repositoryReference}
+      name={obj?.group_value.split('/')[1]}
+      namespace={obj?.group_value.split('/')[0]}
+      /></td>
+      {activeNamespace === ALL_NAMESPACES_KEY && <td className={tableColumnClasses[1]}>{obj?.group_value?.split('/')[0]}</td>}
       <td className={tableColumnClasses[2]}>
-        {obj.summary['runs-in-repositories']}
+        {obj.total}
       </td>
-      <td className={tableColumnClasses[3]}>{obj.summary['total-duration']}</td>
-      <td className={tableColumnClasses[4]}>{obj.summary['avg-duration']}</td>
-      <td className={tableColumnClasses[5]}>{obj.summary['success-rate']}</td>
-      <td className={tableColumnClasses[6]}>{obj.summary['last-runtime']}</td>
+      <td className={tableColumnClasses[3]}>{formatTime(obj.total_duration)}</td>
+      <td className={tableColumnClasses[4]}>{formatTime(obj.avg_duration)}</td>
+      <td className={tableColumnClasses[5]}>{`${Math.round((100 * obj.succeeded) / obj.total)}%`}</td>
+      <td className={tableColumnClasses[6]}>{`${formatTimeLastRunTime(obj.last_runtime)} ago`}</td>
```

Comment:
for running PipelineRuns it shows `- ago` for Last run time. Better to show only -
<img width="710" alt="image" src="https://github.com/openshift-pipelines/console-plugin/assets/2561818/00f4dc10-5afc-447e-b518-6ab24b4bae98">